notebooks/gnn.py

```diff
@@ -0,0 +1,170 @@
+from sklearn.metrics import confusion_matrix
```

Comment:
Esse arquivo é o que mesmo roger? Uma conversão do notebook? Eu acho isso desnecessário estar aqui @JorgeLuizFranco.

Podemos modularizar esse código em outro momento, podemos deixar apenas o notebook pra não ter esse tipo de redundância. Pq se for pra modularizar, tem que colocar no pacote e importar, e não ficar duas cópias, pq aí tem que atualizar em dois lugares. Daí é um saco.

Eu vou deletar aqui, se vocÊ precisar vai estar num commit anterior. Estou fazendo os ajustes pra que esteja ok pra fazer merge, aí eu dou approval, você dá uma última olhada e fazemos merge.

Reply:
Isso, fiz isso só para testar aquele pdm run. Pode retirar por enquanto, acho que só depois que modularizarmos pensamos